scripts/convert.js

```diff
@@ -78,13 +92,13 @@ They do however show the minifed css definition as their comment.
 
 -}
 
-${classes.map(elmify).join('')}`;
+${classes.map(elmify).join('')}`
 
 // writing the string to the file
-fs.writeFile(exportPath, elmString, function(err) {
+fs.writeFile(exportPath, elmString, function (err) {
   if (err) {
-    return console.log(err);
```

Comment:
The rule is simpler than remembering all the times it can cause problems. Plus I use [prettier](https://github.com/prettier/prettier) so it types the semi-colons for me.